src/pages/home/report/ReportActionCompose.js

```diff
@@ -733,6 +850,31 @@ class ReportActionCompose extends React.Component {
                     <ExceededCommentLength commentLength={encodedCommentLength} />
                 </View>
                 {this.state.isDraggingOver && <ReportDropUI />}
+                {!!this.state.suggestedEmojis.length
+            && this.state.shouldShowSuggestionMenu && (
```

Comment:
This formatting is off (again surprised that the linter didn't catch it). Let's please format this block like this:

```jsx
{!_.isEmpty(this.state.suggestedEmojis.length) && this.state.shouldShowSuggestionMenu && (
    <ArrowKeyFocusManager>
        {...}
        {...}
    </ArrowKeyFocusManager>
)}
```